code/espurna/config/general.h

```diff
@@ -1079,6 +1079,10 @@
 #define MQTT_TOPIC_OTA              "ota"
 #define MQTT_TOPIC_TELNET_REVERSE   "telnet_reverse"
 
+// KingArt WiFi Curtain Switch
+#define MQTT_TOPIC_CURTAININ        "curtainin"
+#define MQTT_TOPIC_CURTAINOUT       "curtainout"
```

Comment:
Why not `curtain` for output and `curtain/set` for input?

Reply:
Makes sense changed. /set is alway's added by ESPURNA